src/scene/webscene/index.jsx

```diff
@@ -39,21 +40,29 @@ class Webscene extends Component {
     if (!this.props.view || !this.props.view.map) return;
     this.componentIsMounted = false;
     this.props.view.map.remove(this.state.groupLayer);
+    this.props.view.map.ground.layers.removeMany(this.state.groundLayers);
   }
 
   update(prevProps = {}) {
-    if (!this.props.view || !this.state.groupLayer) return;
```

Comment:
Why can this be removed?

Reply:
it needs to be removed to support webscenes that only have a ground layer but no other layers, right?